xclim/indices/_agro.py

```diff
@@ -297,10 +300,11 @@ def biologically_effective_degree_days(
       Latitude coordinate.
```

Comment:
Should have given this comment on the other PR, but I would specify that lat is not optional when method is in [gladstones, jones], while it is unused with icclim.